### DISCUSSION TITLE
Deepmind AI global weather forecasting

### DISCUSSION THREAD

**Discussion by meb427** [open-meteo/open-meteo#521] [Ideas]
Saw this https://deepmind.google/discover/blog/graphcast-ai-model-for-faster-and-more-accurate-global-weather-forecasting/

Would be interested to have your take on this Patrick. And may be would be interested to check it later to compare with current data. They clairm this would run on normal PC's which could offer a great breakthru.

**Comment by patrick-zippenfenig** [comment]
Hi,

I tried running it on the same day it was published. I see 4 issues:

1. "Normal PC" is a bit far stretched. If run at 0.25° resolution, you need 25 GB memory. In case of GPU computing, 25 GB VRAM are required. I was not able to run the 0.25° version on my RTX 4070 with 12 GB VRAM. Coarser resolutions work, but are less useful. Using CPU instead of GPU is working, but a single forecast hour took so long to calculate that I stopped the process at some point.

2. To actually use it for forecasts, you need real-time initialisation data from ECMWF. Unfortunately, this data is closed source and requires a data subscription from ECMWF at ~80.000 Euro per year. As an alternative, the model could be initialised with GFS or ICON initialisation data (open-data), but the model has been trained on the IFS model and will likely not produce accurate forecasts with a different initialisation. It would be interesting to train the model on past GFS data, but the computational requirements for training seem to be very high and costly.

3. It is not clear to me, if this model will receive any updates in the future. Although the model is now open-source, the [repository](https://github.com/google-deepmind/graphcast) does not seem to be actively maintained. 

4. The model only provides a limited number of weather variables. Any professional application requiring consistent data for solar irradiance or upper level wind, won't benefit from this model.

IIRC, ECMWF has been running the model for more than a year. Ideally, the model output could be published as open-data. I will try to contact ECMWF and ask if the GraphCast model data could be published as CC-BY-4.0.

**Comment by TSellers2** [comment]
In order to preview this model, it is frustrating that you cannot perform any cursor positioning. At a minimum, it would be nice to be able to specify lat/long coordinates.
